### ISSUE TITLE
Minifier fails to build project when mode is set to "full"

### ISSUE THREAD

**Issue by feoranis26** [malware-dev/MDK-SE#328]
For some reason the minifier is unable to build this specific class (and some other classes from my script, but this minimal example should have the same issue).

I don't know what it is that is causing this specific class to fail, I removed everything except only the involved members to pinpoint the issue, but it builds fine like that, and I think looking at the minifier output is neccessary to see just what went wrong. It seems to skip some members of the class on symbol renaming so the output doesn't match the renamed symbols.

Output log:
```
Build started at 18:06...
1>------ Build started: Project: mdktestscript, Configuration: Debug x64 ------
1>Found local ini file: C:\Users\alpdo\source\repos\mdktestscript\mdktestscript\mdktestscript.mdk.local.ini
1>Found ini file: C:\Users\alpdo\source\repos\mdktestscript\mdktestscript\mdktestscript.mdk.ini
1>Successfully determined the binary path of Space Engineers: c:\program files (x86)\steam\SteamApps\common\SpaceEngineers\Bin64
1>Loading Space Engineers assemblies from c:\program files (x86)\steam\SteamApps\common\SpaceEngineers\Bin64
1>MDK is restoring ingame script project: mdktestscript
1>mdktestscript -> C:\Users\alpdo\source\repos\mdktestscript\mdktestscript\bin\x64\Debug\netframework48\mdktestscript.dll
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(43,152): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(44,24): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(44,54): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(44,80): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(44,105): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(45,10): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(45,34): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(45,57): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(45,83): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(45,109): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(46,9): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(46,44): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(46,68): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(46,107): error CS0103: The name 'Assign' does not exist in the current context
1>C:\Users\alpdo\AppData\Roaming\SpaceEngineers\IngameScripts\local\mdktestscript\script.cs(47,21): error CS0103: The name 'Assign' does not exist in the current context
1>Failed to compile the project.
1>C:\Users\alpdo\.nuget\packages\mal.mdk2.pbpackager\2.0.2\build\Mal.Mdk2.PbPackager.props(18,9): error MSB3073: The command ""C:\Users\alpdo\.nuget\packages\mal.mdk2.pbpackager\2.0.2\build\..\tools\mdk.exe" pack "C:\Users\alpdo\source\repos\mdktestscript\mdktestscript\mdktestscript.csproj"  -configuration Debug -interactive" exited with code -2.
1>Done building project "mdktestscript.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
========== Build completed at 18:06 and took 03.763 seconds ==========
```
[issue.zip](https://github.com/user-attachments/files/15916101/issue.zip)

**Comment by feoranis26** [comment]
possibly a duplicate of #325

**Comment by malware-dev** [comment]
Fixed in 2.0.4
